src/pages/PostPage/PostPage.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { useState } from 'react'
+import * as S from './PostPage.style.js';
+import Button from '../../components/MainButton';
+import SelectedImg from '../../assets/icon/background-selected.png'
+
+const PostPage = () => {
+  const [choiceColor, setChoiceColor] = useState(true);
+  const [choiceImage, setChoiceImage] = useState(false);
+
+  //컬러 버튼을 눌렀을때 => 컬러버튼 활성화, 이미지버튼 비활성화
+  const handleBackgroundColor = () => {
+    setChoiceColor(true);
+    setChoiceImage(false);
```

Comment:
```suggestion
    setBackgrounType('color');
```

위를 바꾸면 이런식으로 관리할 수 있습니다.